test-js/integration.js

```diff
@@ -192,6 +192,13 @@ describe("integration tests", function() {
     assert.exists(path.join("output", "Main", "index.js"));
   }));
 
+  notWindowsIt("pulp build with psc-package", run(function*(sh, pulp, assert, temp) {
```

Comment:
There just isn't a Windows binary in the psc-package releases, but I just tried building it on Windows and it works fine.